datafusion/physical-plan/src/metrics/baseline.rs

```diff
@@ -58,6 +58,7 @@ pub struct BaselineMetrics {
 impl BaselineMetrics {
     /// Create a new BaselineMetric structure, and set `start_time` to now
     pub fn new(metrics: &ExecutionPlanMetricsSet, partition: usize) -> Self {
+        // TODO: Is start_time used anywhere?
```

Comment:
Drive-by: Is `start_time` actually made use of? Otherwise, I suggest dropping it.